internal/controller/postgrescluster/controller.go

```diff
@@ -204,11 +204,7 @@ func (r *Reconciler) Reconcile(
 		})
 		return patchClusterStatus()
 	} else {
-		// Avoid a panic! Fixed in Kubernetes v1.21.0 and controller-runtime v0.9.0-alpha.0.
```

Comment:
If we're supporting K8s 1.20, then maybe we should put these guards back

Reply:
OK, I've run check-envtest-existing and check-kultt pointed at 4.6 and had no problems related to this issue. (There were other problems, but I didn't see any panics.)

Reply:
That's good :+1:. Might be worth discussing, but if it passes in OCP 4.6, we should be in good shape since 'vanilla kube' should be 1.21+ at this point (per the [docs](https://access.crunchydata.com/documentation/postgres-operator/v5/)).